### PR TITLE
Update Highest/Lowest scoring Away/Home team methods with correct math

### PR DESCRIPTION
**Pull request by KirbyDD**
Test suite was updated to account for new calculation
Methods pass Test Suite
Methods pass Rspec

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -0,0 +1 @@
+--require spec_helper
```
Comment (hale4029) [inline .rspec]: delete file

File: coverage/.last_run.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 58.27
```
Comment (hale4029) [inline coverage/.last_run.json:3]: delete file, we haven't been  pushing the coverage stuff

File: coverage/.resultset.json [hunk 3]
```diff
@@ -270,64 +270,78 @@
         1,
```
Comment (hale4029) [inline coverage/.resultset.json]: delete file from pr

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -0,0 +1,100 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```
Comment (hale4029) [inline spec/spec_helper.rb]: delete file

**Review by hale4029 — CHANGES_REQUESTED** [review]
looks good, i'd just remove the coverge files and rspec files from the merge/PR

File: .rspec (hunk 1 above)
Comment (KirbyDD) [reply .rspec]: will do

Comment (KirbyDD) [reply .rspec]: Removed

File: coverage/.last_run.json (hunk 2 above)
Comment (KirbyDD) [reply coverage/.last_run.json:3]: removed

File: coverage/.resultset.json (hunk 3 above)
Comment (KirbyDD) [reply coverage/.resultset.json]: removed

File: spec/spec_helper.rb (hunk 4 above)
Comment (KirbyDD) [reply spec/spec_helper.rb]: removed

**Review by msimon42 — APPROVED** [review]
Looks good. Thanks for changing those tests.

**Comment by msimon42** [comment]
Looks good. I would consider changing that new_hash variable name to something more descriptive.

**Pull request merged**